lib/pronto/config_file.rb

```diff
@@ -35,8 +36,10 @@ class ConfigFile
       'format' => DEFAULT_MESSAGE_FORMAT
     }.freeze
 
-    def initialize(path = '.pronto.yml')
-      @path = path
+    def initialize
+      @path = ENV.fetch('PRONTO_CONFIG', DEFAULT_FILE_PATH)
+      return if @path == DEFAULT_FILE_PATH || File.exist?(@path)
```

Comment:
Is this check really necessary?  If the custom file is missing or the default is missing, we should still raise an error.